packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
Hmm. Maybe you're right, since indeed there's no yielding in Minimongo.

Unfortunately I won't be able to look that much more deeply at this right now.

Modifying the instance object when you run a method and then reverting those changes still smells wrong to me. It could make debugging harder, for example. Why would we do it that way?

As for the upgrade path -- it should just make sure to make users aware that `count()` on the client now behaves differently. So if they pass no argument to `count()` it should print a warning (or perhaps even an error, not sure about that one)